doc/INSTALL-MAC.md

```diff
@@ -168,7 +168,7 @@ Run the following commands to build libopenshot:
 $ cd [libopenshot repo folder]
 $ mkdir build
 $ cd build
-$ cmake -G "Unix Makefiles"  -DCMAKE_CXX_COMPILER=/usr/local/opt/gcc48/bin/g++-4.8 -DCMAKE_C_COMPILER=/usr/local/opt/gcc48/bin/gcc-4.8 -DCMAKE_PREFIX_PATH=/usr/local/Cellar/qt5/5.4.2/ -DPYTHON_INCLUDE_DIR=/usr/local/Cellar/python3/3.3.2/Frameworks/Python.framework/Versions/3.3/include/python3.3m/ -DPYTHON_LIBRARY=/usr/local/Cellar/python3/3.3.2/Frameworks/Python.framework/Versions/3.3/lib/libpython3.3.dylib -DPython_FRAMEWORKS=/usr/local/Cellar/python3/3.3.2/Frameworks/Python.framework/ ../ -D"CMAKE_BUILD_TYPE:STRING=Debug"
+$ cmake --debug-output -G "Unix Makefiles"  -DCMAKE_CXX_COMPILER=/usr/local/opt/gcc48/bin/g++-4.8 -DCMAKE_C_COMPILER=/usr/local/opt/gcc48/bin/gcc-4.8 -DCMAKE_PREFIX_PATH=/usr/local/Cellar/qt5/5.4.2/ -DPYTHON_INCLUDE_DIR=/usr/local/Cellar/python3/3.3.2/Frameworks/Python.framework/Versions/3.3/include/python3.3m/ -DPYTHON_LIBRARY=/usr/local/Cellar/python3/3.3.2/Frameworks/Python.framework/Versions/3.3/lib/libpython3.3.dylib -DPython_FRAMEWORKS=/usr/local/Cellar/python3/3.3.2/Frameworks/Python.framework/ ../ -D"CMAKE_BUILD_TYPE:STRING=Debug"
```

Comment:
Yeah, this is actually all wrong. It's a very bad idea to just randomly switch between GCC and clang when compiling interoperable libraries, for starters.

```suggestion
$ cmake -G "Unix Makefiles"  -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_C_COMPILER=clang -DCMAKE_PREFIX_PATH="$(brew --cellar qt5)/<version>" -DPython_FRAMEWORKS="$(brew --cellar python3)/<version>/Frameworks/Python.framework/" ../ -D"CMAKE_BUILD_TYPE:STRING=Debug"
```

Unfortunately there doesn't seem to be a generic `brew` command to look up a _specific_ version's Cellar directory, but that's mostly because you're really not supposed to have to do that. 

I'd be surprised if Homebrew CMake can't find Homebrew Python _without_ needing any paths specified, or at most by adding something like the root `$(brew --cellar)` directory to `CMAKE_PREFIX_PATH`. But I don't have a Mac to test that on. I do know that they avoid installing Python into the default-accessible paths to avoid conflicts with system packages, which makes sense, but CMake can handle that if properly configured.

I'm **certain** there's a better way to go about this than the instructions we've been providing here, though, which were based on truly _ancient._ versions of CMake and don't take advantage of any of its more modern discovery features.

Reply:
Also, I'd suggest to remove the preceding `$ ` characters from all of these lines, again to facilitate copy-pasting.